client/reader/user-stream/components/user-profile-header/style.scss

```diff
@@ -1,31 +1,95 @@
+@import '@wordpress/base-styles/mixins';
+@import '@wordpress/base-styles/breakpoints';
+
 .is-section-reader {
 	.user-profile-header {
 		max-width: 768px;
 		margin: 0 auto;
+		padding: 0 16px;
 
 		&__main {
-			align-items: center;
 			display: flex;
 			padding: 36px 0 24px;
+			align-items: center;
 
-			.reader-avatar {
-				margin: 0 24px 0 0;
+			@include breakpoint-deprecated( '<660px' ) {
+				padding-top: 20px;
 			}
 		}
 
+		&__avatar {
+			margin-right: 24px;
+			flex-shrink: 0;
+			width: 116px;
+			height: 116px;
+
+			img {
+				border-radius: 100%;
+				width: 100%;
+				height: 100%;
+			}
+
+			&-mobile {
+				width: 72px;
+				height: 72px;
+				margin-right: 16px;
+				display: block;
+
+				@include break-mobile {
+					display: none;
+				}
+			}
+
+			&-desktop {
+				display: none;
+
+				@include break-mobile {
+					display: block;
+					margin-right: 24px;
+					width: 116px;
+					height: 116px;
+				}
+			}
+		}
+
+		&__details {
+			flex: 1;
+			min-width: 0;
```

Comment:
Do we need the `min-width` value here? I've checked and it doesn't seem necessary.

Reply:
It's helpful for really long text. For example, a long name will prevent the flexbox from shrinking past a certain width due to it sizing to the content, and then the bio will also run off the screen with the name. This prevents that.